features/projects/components/project-list/project-list.tsx

```diff
@@ -27,7 +39,17 @@ export function ProjectList() {
 
   if (isError) {
     console.error(error);
-    return <div>Error: {error.message}</div>;
+    return (
+      <Badge color={BadgeColor.error} size={BadgeSize.lg}>
```

Comment:
I would use a simple `div` instead of the `Badge` component here. The badge is meant to be a rather small element used for tags and such. But  I get how they look similar.

This could be something like this

```
styled.div`
  display: flex;
  padding: 1rem;
  background: #FFFBFA;
  border: 1px solid #FDA29B;
  border-radius: 8px;
`
```

or even better with the existing `color` and `space` utility functions

```
styled.div`
  display: flex;
  padding: ${space(4)};
  background: ${color("error", 25)};
  border: 1px solid ${color("error", 300);
  border-radius: 8px;
`
```